pipeline/schemas/user.py

```diff
@@ -18,12 +18,15 @@ class UserBase(AvatarHolder):
 
 class UserGet(UserBase):
     id: str
-    base_token: TokenGet
     oauth_provider: Optional[str]
     verified: Optional[bool]
     subscribed: Optional[bool]
 
 
+class EnrichedUserGet(UserGet):
+    base_token: TokenGet
```

Comment:
Unfortunately the [naming conventions](https://github.com/neuro-ai-dev/pipeline_schemas#naming-conventions) didn't survive the schema migration but `UserGetEnriched` would be more consistent with other 'extended' names. I wouldn't be against something more explicit here e.g. `UserGetWithBaseToken`.

Reply:
I thought about making it specific, but then erinched felt it delivers the idea and allows for more data to be added in the future, I switch around the wording!